schemas/JSON/settings/settings.schema.0.2.json

```
@@ -205,7 +205,8 @@
         "defaultModuleRoot": {
           "description": "The default root directory where PowerShell modules are installed to when applying a configuration.",
           "type": "string",
-          "maxLength": 32767
+          "maxLength": 32767,
+          "default": "%LOCALAPPDATA%/Microsoft/Winget/Configuration/Modules/"
```

```suggestion
          "default": "%LOCALAPPDATA%/Microsoft/WinGet/Configuration/Modules/"
```

Though Windows file-system is case-insensitive, this should match the casing used in other paths for consistency

---

Got it @mdanish-kh.